services/minespace-web/src/components/Forms/projects/majorMineApplication/MajorMineApplicationForm.tsx

```diff
@@ -0,0 +1,382 @@
+import React, { useState } from "react";
+import { Link } from "react-router-dom";
+import { useDispatch, useSelector } from "react-redux";
+import { Field, change, getFormValues } from "redux-form";
+import "@ant-design/compatible/assets/index.css";
+import { Col, Row, Typography, Button } from "antd";
+import { required } from "@common/utils/Validate";
+import {
+  MAJOR_MINES_APPLICATION_DOCUMENT_TYPE,
+  MAJOR_MINES_APPLICATION_DOCUMENT_TYPE_CODE,
+} from "@mds/common/constants/strings";
+import { DOCUMENT, Feature, MODERN_EXCEL, SPATIAL, SPATIAL_DATA_STANDARDS_URL } from "@mds/common";
+import * as routes from "@/constants/routes";
+import * as FORM from "@/constants/forms";
+import { renderConfig } from "@/components/common/config";
+import { documentNameColumn, uploadDateColumn } from "@/components/common/DocumentColumns";
```

Comment:
Would be nice to update a bunch of these imports to the @mds/common version where possible.. Like from Validate, FORM, DocumentColumns, and DocumentTable (I deleted that one on my branch!)  ArchivedDocumentsSection. Combine the imports from the same source (I see two from strings, and get rid of the legacy antd stylesheet.

Would also love to see renderConfig taken out and RenderField put in so that we're a little closer to being able to delete the old ones.